src/components/directory-content.js

```diff
@@ -719,60 +753,37 @@ const DirectoryContent = () => {
         setSelectedUuids(new Set());
     }, [handleError, currentChildren, currentChildrenRef]);
 
-    const contextualMixPolicies = {
-        BIG: 'GoogleMicrosoft', // if !selectedUuids.has(selected.Uuid) deselects selectedUuids
-        ZIMBRA: 'Zimbra', // if !selectedUuids.has(selected.Uuid) just use activeElement
-        ALL: 'All', // union of activeElement.Uuid and selectedUuids (actually implemented)
-    };
-    let contextualMixPolicy = contextualMixPolicies.ALL;
-
-    const getSelectedChildren = (mayChange = false) => {
-        let acc = [];
-        let ctxtUuid = activeElement ? activeElement.elementUuid : null;
-        if (activeElement) {
-            acc.push(
-                Object.assign(
-                    {
-                        subtype:
-                            childrenMetadata[activeElement.elementUuid]
-                                ?.subtype,
-                    },
-                    activeElement
-                )
-            );
-        }
-
-        if (selectedUuids && currentChildren) {
-            if (
-                contextualMixPolicy === contextualMixPolicies.ALL ||
-                ctxtUuid === null ||
-                selectedUuids.has(ctxtUuid)
-            ) {
-                acc = acc.concat(
+    const getSelectedChildren = () => {
+        let selectedChildren = [];
+        if (currentChildren?.length > 0) {
+            // Adds the previously selected elements
+            if (selectedUuids?.size) {
+                selectedChildren = selectedChildren.concat(
```

Comment:
With what do you concat ? an empty array ? maybe this form is enought :
```
let selectedChildren = currentChildren.filter(...).map(...)
```

Reply:
Yes, nice catch